### PR TITLE
[46] Product Page using HTML

### PR DESCRIPTION
**Pull request by Goldochim**
## Task: `[46] Product Page using HTML`
- `I re-edited my html file and uploaded the screenshots of the display in a browser'. I also included the images used in the web page

`
Screenshot (14)
Screenshot (15)

headp3
headp5

### CODE REVIEW

File: submissions/goldOchim/aboutMe/HTML/index.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    <body>
+        <h1>
+            GA-JETS STORES 
+        </h1>
+           
+            <table>
+               
+                <tr>
+                    <th>  <img src="headp5.jpeg">
+                    </img> </th>
```
Comment (adaugochi) [inline submissions/goldOchim/aboutMe/HTML/index.html:18]: `</img>` is not used properly

File: submissions/goldOchim/aboutMe/HTML/index.html [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+                    <th>Beats by Dre HeadPhones </th>
+                    <th>N97,000</th>
+                    <th>     Bass sound system, sensor sensitive control, fitting head grip</th>
+                </tr>
+                <tr>
+                    <th>  <img src="headp3.jpeg">
+                    </img> </th>
```
Comment (adaugochi) [inline submissions/goldOchim/aboutMe/HTML/index.html:25]: same here

**Review by adaugochi — CHANGES_REQUESTED** [review]
Close the other PR you opened if not in use

File: submissions/goldOchim/aboutMe/HTML/index.html (hunk 1 above)
Comment (Goldochim) [reply submissions/goldOchim/aboutMe/HTML/index.html:18]: Is it the bad coding practice of where it is placed or something else. I don't really understand.

Comment (adaugochi) [reply submissions/goldOchim/aboutMe/HTML/index.html:18]: img tag is a self-closing tag. It does not have a closing tag

**Review by adaugochi — CHANGES_REQUESTED** [review]
Also, the images you used for the task, I can't find it as part of your PR

File: submissions/goldOchim/aboutMe/HTML/index.html (hunk 1 above)
Comment (Goldochim) [reply submissions/goldOchim/aboutMe/HTML/index.html]: Oh, okay. I have made the corrections and attached the images.

**Review by adaugochi — CHANGES_REQUESTED** [review]
Hello

I can't find the images you used for this task as part of your PR

**Comment by Goldochim** [comment]
> Hello
> 
> I can't find the images you used for this task as part of your PR



> Hello
> 
> I can't find the images you used for this task as part of your PR

I later attached it alongside the screen shots.

**Comment by adaugochi** [comment]
@Goldochim It has to be part of your PR, not just as a screenshot. Simply add it as part of your PR

**Review by ToluAkin — CHANGES_REQUESTED** [review]
@Goldochim The images used on this page is not added to the pull request. You need to update that.

**Review by Goldochim — COMMENTED** [review]
headp3
headp5
Screenshot (15)
Screenshot (14)

**Review by ToluAkin — APPROVED** [review]

**Review by ToluAkin — DISMISSED** [review]

**Comment by ToluAkin** [comment]
@Goldochim Are you having issues updating the pull request? If you have any issue do reach out

File: submissions/goldOchim/aboutMe/HTML/index.html [hunk 3]
```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
```
Comment (adaugochi) [inline submissions/goldOchim/aboutMe/HTML/index.html:1]: This file is inside the wrong folder. The `HTML` folder shouldn't be inside `aboutMe` folder. instead, it should be inside your name folder

**Review by adaugochi — CHANGES_REQUESTED** [review]

File: submissions/goldOchim/aboutMe/HTML/index.html (hunk 3 above)
Comment (adaugochi) [reply submissions/goldOchim/aboutMe/HTML/index.html:1]: Are you having an issue updating this change?

**Review by adaugochi — CHANGES_REQUESTED** [review]

**Comment by adaugochi** [comment]
Long over due

**Pull request closed without merging**